### DISCUSSION TITLE
Plus code search doesn't find anything

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#363] [Q&A] [answered]
It was corrected already in the past, but in B15 it seems the the plus code search again takes ages or doesn't find anything, even if maps and pois for this country is active, example "646V+WF Corgémont, Schweiz" nothing is found.

**Comment by devemux86** [comment, marked as answer]
This is a local plus code with a locality to search online.

The global plus code is: https://plus.codes/8FV9646V+WF
With the short-form "646V+WF Corgémont, Switzerland".

So it depends on whether the selected online search service can find the plus code's locality.

- OpenRouteService doesn't find the "Corgémont, Schweiz", but it finds the "Corgémont, Switzerland"
- GraphHopper / MapQuest can find the "Corgémont, Schweiz"

<table>
<tr>
<th>OpenRouteService</th><th>GraphHopper / MapQuest</th>
</tr>
<tr>
<td width="50%"><img src="https://github.com/devemux86/cruiser/assets/3484020/b4abb56c-4202-4ad7-ab5c-94ea467db564"/></td><td><img src="https://github.com/devemux86/cruiser/assets/3484020/422d0f0d-5354-4ff4-ac9e-7b4098383851"/></td width="50%">
</tr>
</table>

**Reply by DolomitesTom** [reply]
It looks like Graphhopper finds it much quicker. I have switched now to Graphhopper search engine and will test it further. Thanks for the hint.
